spec/app/dumpers/z_after_hook_dumper.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+FILENAME = 'tmp/test.out'
+
+# Filename prepended with z to so it is run last
```

Comment:
I think this further goes to suggest that this test should be separate.

Reply:
I agree that it would be best as a separate test. Because there's no way to run dumpers individually only way I could think to do that was to create a second test app, which I thought might be overkill. That said, I do think it'd be cleaner than what I have here. What are your thoughts on adding a second test app? Also open to any other suggestions.

Reply:
> Because there's no way to run dumpers individually

I think this would be a welcome feature if that is the best approach to the problem. It would allow a better workflow when only a single dumper changes. But this is not required for this feature to land.

> What are your thoughts on adding a second test app?

I'm okay with this approach as well. We should name the apps such that their purpose is clear.